data/pages.json

```diff
@@ -97,5 +98,10 @@
         "description": "Autumn 404",
         "imageUrl": "/__screenshots/autumn.png?raw=true",
         "url": "autumn/404.html"
+    },
+    {
+        "description": "Imposter 404",
+        "imageUrl": "/__screenshots/imposter.png?raw=true",
```

Comment:
Please keep the `imageUrl` format from other templates.

Reply:
Please Review @matteobruni